src/components/CaseStudies/CaseStudyContentContainer.tsx

```diff
@@ -11,10 +10,7 @@ export function CaseStudyContentContainer({
 }: {
   children: React.ReactNode
   className?: string
-  currentProject: {
-    pv: PV
-    id: number
-  }
+  currentProject: string
```

Comment:
`currentProject` is now just the `id` of a project (i.e. "1-17")

previously the `currentProject.id` was actually the project's `projectId` which was misleading. because we can now query projects with a list of `id`s via the api, passing `projectId` and `pv` was unnecessary extra complexity